seismicpro/batch.py

```diff
@@ -476,6 +476,18 @@ def calculate_metric(self, metric, *args, metric_name=None, coords_component=Non
         self._num_calculated_metrics += 1
         return self
 
+    @staticmethod
+    def _unpack_args(batch_item, args):
```

Comment:
```suggestion
    def _unpack_args(args, batch_item):
```
More clear as for me